src/manager/kiwix-manage.cpp

```diff
@@ -182,7 +182,7 @@ int handle_add(kiwix::Library* library, const std::string& libraryPath,
   return(resultCode);
 }
 
-int handle_remove(kiwix::Library* library, const std::string& libraryPath,
+int handle_remove(kiwix::LibraryPtr library, const std::string& libraryPath,
```

Comment:
I think that a parameter of type `kiwix::Library&` is more suitable here (passing a `shared_ptr` suggests that it is preserved somewhere and can be accessed after the function returns).

Reply:
Agree.